frontend/dockerfile/linter/ruleset.go

```diff
@@ -15,10 +15,10 @@ var (
 	}
 	RuleFromAsCasing = LinterRule[func(string, string) string]{
 		Name:        "FromAsCasing",
-		Description: "The 'as' keyword should match the case of the 'from' keyword",
+		Description: "The AS keyword should match the case of the FROM keyword",
 		URL:         "https://docs.docker.com/go/dockerfile/rule/from-as-casing/",
 		Format: func(from, as string) string {
-			return fmt.Sprintf("'%s' and '%s' keywords' casing do not match", as, from)
+			return fmt.Sprintf("'%s' and '%s' keywords casing do not match", as, from)
```

Comment:
This would be grammatically incorrect. Keywords' is possessive.

Reply:
Alternatively:

```
return fmt.Sprintf("Casing mismatch: keywords '%s' and '%s' should match in case", from, as)
```

Reply:
Thank you! And I like your suggestion.